show/main.py

```diff
@@ -62,13 +60,10 @@ def __init__(self):
             raise click.Abort()
 
         for port_name in self.port_dict.keys():
-            try:
```

Comment:
Why are you removing this try/except block?

Reply:
My bad. I thought I had grabbed the most recent version of the file but it looks like I missed a step.   I've updated this now.